x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_exceptions/components/add_exception_flyout/index.tsx

```diff
@@ -536,6 +541,7 @@ export const AddExceptionFlyout = memo(function AddExceptionFlyout({
         {errorSubmitting != null && (
           <>
             <EuiCallOut
+              announceOnMount
```

Comment:
Is this related to a bug, or just a best practice? I can [see](https://eui.elastic.co/docs/components/display/callout/#screen-reader-announcements) what it does, but it's not clear why we're adding it now.